easybuild/easyconfigs/g/GROMACS/README.md

```diff
@@ -104,11 +108,20 @@ engines.
     of enabling support for the `cray-python` modules. Therefore the corresponding 
     EasyConfigs of GROMACS have also been replaced.**
 
+### GROMACS 2023.2 with AMD GPU support for CPE 22.12
+
+  * There are multiple easyconfig files for running GROMACS with AMD GPU acceleration, that follows [installation guide](https://manual.gromacs.org/2023.2/install-guide/index.html#sycl-gpu-acceleration-for-amd-gpus).
+
+    * [VkFFT enabled](https://manual.gromacs.org/2023.2/install-guide/index.html#using-vkfft) version targetting single node execution with tMPI and multi GPU offloading
+    * [HeFFTe enabled](https://manual.gromacs.org/2023.2/install-guide/index.html#using-heffte) version targetting multinode, multi GPU execution with MPI and hipSYCL
+    * Generic hipSYCL version with GPU support enabled
```

Comment:
Hi! This description is technically correct, but is a bit misleading: 

1. VkFFT is the default choice for hipSYCL build, so the difference between "VkFFT enabled" and "Generic" version is in the MPI implementation, not FFT.

2. "version targetting single node execution with tMPI and multi GPU offloading": our SYCL backend does not support peer-to-peer communications, so the threadMPI version is unlikely to ever be more performant than the one with proper GPU-aware MPI, even on a single node.

So, overall, it feels like "Generic" and "HeFFTe enabled" versions are enough. The "VkFFT enabled" does not add much.

Reply:
Thanks for pointing it out. This sentence could use some work. "it is the default" would be more precise.